src/modules/lists/dto/list.dto.ts

```diff
@@ -16,6 +16,11 @@ export class ListDto {
   @IsBoolean()
   readonly public: boolean;
 
+  @ApiModelPropertyOptional()
+  @IsOptional()
+  @IsBoolean()
```

Comment:
Please remove these decorators, we don't want to allow clients to update this property. There should always be a single `isFavorite=true` list in for each user, and that list gets created automatically, therefore we need to make sure clients can't update this property

Reply:
Cool I will go ahead and remove this property here. Come to think of it I don't think it's necessary in ListDto